src/main/resources/com/arcbees/bourseje/client/resources/css/pages/candidates.gss

```diff
@@ -0,0 +1,42 @@
+.text_intro {
+    margin-bottom: 100px;
+}
+
+.text_intro h2,
```

Comment:
Try to use `>` when possible, especially when your rightmost element (the key selector) is too general.